trino-connector/src/main/java/com/datastrato/gravitino/trino/connector/GravitinoColumnHandle.java

```diff
@@ -1,64 +0,0 @@
-/*
- * Copyright 2023 Datastrato Pvt Ltd.
- * This software is licensed under the Apache License version 2.
- */
-package com.datastrato.gravitino.trino.connector;
-
-import static com.google.common.base.MoreObjects.toStringHelper;
-
-import com.fasterxml.jackson.annotation.JsonCreator;
-import com.fasterxml.jackson.annotation.JsonProperty;
-import com.google.common.base.Preconditions;
-import io.trino.spi.connector.ColumnHandle;
-
-/**
- * The GravitinoColumnHandle is used to transform column information between Trino and Gravitino, as
- * well as to wrap the inner connector column handle for data access.
- */
-public final class GravitinoColumnHandle implements ColumnHandle {
```

Comment:
Why are we removing this class? you'd better add some description about this point.

Reply:
The query optimizer iterates continuously during the query optimization process. Initially, it invokes the GravitionMetadata interface using GravitinoTableHandle and GravitinoColumnHandle as parameters. After processing by the internal connector, the returned optimization results may include JDBCTableHandle and JDBCColumnHandle. As the optimization continues, the parameters passed to the GravitionMetadata interface could be a mix of GravitinoTableHandle and JDBCColumnHandle, or JDBCTableHandle and GravitinoColumnHandle, or even just JDBCTableHandle and JDBCColumnHandle. Therefore, the previous code contained a significant amount of logic for checking the types of TableHandle and ColumnHandle. This update removes GravitinoColumnHandle to ensure that the ColumnHandle used by GravitionMetadata is JDBCColumnHandle, thereby avoiding type conversions and scenarios where conversion is not possible.

Reply:
> This update removes GravitinoColumnHandle to ensure that the ColumnHandle used by GravitionMetadata is JDBCColumnHandle

So, when we query tables backed by Iceberg catalogs,  the implementation of `ColumnHandle` is also `JDBCColumnHandle`?

Reply:
No use mysql connector is the `JDBCColumnHandle`